vignettes/sample.txt

```diff
@@ -7,6 +7,7 @@ github: mitchelloharawild
 linkedin: mitchelloharawild
 twitter: mitchoharawild
 headcolor: 414141
+footer_docname:: CV/Resume
```

Comment:
I'm not sure if footer_docname is the best name for this. It is certainly descriptive for the awesomecv template, however elements in the YAML header should apply to all templates, which may include similar text in places other than the footer. Perhaps `docname` is better?

Also only one colon is required here.
```suggestion
footer_docname: CV/Resume
```

Reply:
You mentioned that in the email. I haven't looked at the .tex for the other templates, but they don't look (to me) like that would be used for resume's. But I didn't completely think it through how the YAML is the same across templates. I think that template specific YAML samples would be helpful for others so they can understand elements that do/do not apply.